app/services/rapid_ready_for_decision/diagnostic_codes.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module RapidReadyForDecision
+  module DiagnosticCodes
+    HYPERTENSION = 7101
+  end
+end
```

Comment:
Out of curiosity what was the reasoning for making this its own module? Is this a pattern for dealing with constants?

Reply:
There wasn't an existing class AFAIK that made sense where I could put a constant like:
```ruby
DIAGNOSTIC_CODES = {
  hypertension: 7101
}
```
It's used for various classes in RRD, so I ended up putting the constant in its own file. I'm open to suggestions.